model.cc

```diff
@@ -11,7 +11,7 @@ using namespace std;
 
 int main() {
 
-  DataSet data_set("/Users/fco/Documents/personnel/projects/tensorflow/tensorflow/cc/models/normalized_car_features.csv");
+  DataSet data_set("normalized_car_features.csv");
```

Comment:
I guess this will only be valid in my case. I copy the file into my binary folder

Reply:
Yeah, I am wondering how to do it using bazel.

Reply:
FYI I am still searching a way to copy the file in the binary dir.

Reply:
Finally found a way to do it with bazel, should be merged soon.